ovl_util/widgets.py

```diff
@@ -706,7 +706,8 @@ def setColor(self, color):
 			self.colorChanged.emit(color)
 
 		if self._color:
-			self.setStyleSheet("background-color: %s;" % self._color.name(QtGui.QColor.NameFormat.HexArgb))
+			color_hex = self._color.name(QtGui.QColor.NameFormat.HexArgb)
+			self.setStyleSheet(f"QColorButton {{background-color: {color_hex}; border: none; min-width: 100px; min-height: 18px; border-radius: 3px;}}")
```

Comment:
Also, this stylesheet was also applying to the entire color picker window before, setting its background color.  If this was a "feature" it can be added back.

Reply:
That was a known issue, not intended